packages/contracts/docs/osx/02-how-to-guides/01-dao/01-best-practices.md

```diff
@@ -2,23 +2,21 @@
 title: Best Practices
 ---
 
-## Advices for Operating your DAO
+## Some Advice When Operating your DAO
 
 ### DOs 👌
 
-- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that it can execute on the DAO.
-- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review
+- Make sure that at least one address (typically a governance plugin) has `EXECUTE_PERMISSION_ID` permission so that something can be execute on behalf of the DAO.
+- Check every proposal asking to install, update, or uninstall a plugin with utmost care and review. Installation means granting an external contract permissions, so you want to be extra careful about:
   - the implementation contract
   - the setup contract
   - the helper contracts
   - the permissions
 
 ### DON'Ts ✋
 
-- Incapacitate your DAO by revoking the last `EXECUTE_PERMISSION_ID`.
-  - by uninstalling your last governance plugin.
+- Incapacitate your DAO by revoking the last `EXECUTE_PERMISSION_ID`. This means your DAO will be blocked and any assets you hold may be locked in forever. Revoking `EXECUTION_PERMISSION_ID` is done through:
+  - uninstalling your last governance plugin.
   - applying an update to your last governance plugin.
-- Creating a DAO with no plugins, or with plugins that can’t effectively execute.
-- Giver permissions to directly call things on the DAO. Better use a plugin instead.
-
-In the following sections, you will learn about the details.
+- You shouldn't create a DAO with no plugins, or with plugins that can’t effectively execute. This will block your DAO from ever being used, since you need some kind of governance mechanism to execute actions.
```

Comment:
"since you need some kind of governance mechanism to execute actions."
This statement is not true. A caller (e.g., a wallet) can directly execute actions without a governance mechanism if it has the `EXECUTE_PERMISSION` granted to it.

I would leave this part out.

Reply:
Nice! Will edit.